cognite/client/stable/datapoints.py

```diff
@@ -441,7 +441,7 @@ def post_datapoints(self, name, datapoints: List[Datapoint]) -> None:
         Args:
             name (str):       Name of timeseries to insert to.
 
-            datapoints (list[stable.datapoints.Datapoint): List of datapoint data transfer objects to insert.
+            datapoints (list[stable.datapoints.Datapoint]): List of datapoint data transfer objects to insert.
```

Comment:
should actually be List with capital L as well. https://docs.python.org/3/library/typing.html#typing.List